spec/services/courses/creation_service_spec.rb

```diff
@@ -19,6 +19,27 @@
 
   let(:next_available_course_code) { false }
 
+  context 'when teacher degree apprenticeship course' do
+    let(:valid_course_params) do
+      {
+        'level' => 'primary',
+        'is_send' => '1',
+        'age_range_in_years' => '3_to_7',
+        'qualification' => 'undergraduate_degree_with_qts'
+      }
+    end
+
+    it 'creates the teacher degree apprenticeship course' do
+      expect(subject.program_type).to eq('teacher_degree_apprenticeship')
+      expect(subject.funding_type).to eq('apprenticeship')
+      expect(subject.can_sponsor_student_visa?).to be false
+      expect(subject.can_sponsor_skilled_worker_visa?).to be false
+      expect(subject.additional_degree_subject_requirements).to be_nil
+      expect(subject.degree_subject_requirements).to be_nil
```

Comment:
Can we remove from the test too please 🙏🏼

Reply:
Humm, I left on purpose, I think the tests should be there to guarantee any new bugs to be introduced (because this field should be nil regardless of scenario in TDA).

Reply:
Example:

Someone wake up in a morning 2 years from now and decided the DB default to this field to be true. Suddenly we have a bug on TDA and the tests should alert us about it.

Reply:
Ok, makes sense! All good in that case. 

I did wonder if it might be worth explicitly mentioning that in a comment but I think it's self explanatory, re-reading the test.